`grz-check` fails to compile

Hi,

I'm using `grz-check` from cargo. Updating to version 0.2.0 fails because of a build error:

```
> cargo install --locked --root /usr/local grz-check --version 0.2.0
   Compiling zstd v0.13.3
   Compiling niffler v3.0.0
   Compiling grz-check v0.2.0
error[E0658]: `let` expressions in this position are unstable
   --> /root/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/grz-check-0.2.0/src/checker.rs:481:8
    |
481 |     if let Ok(mutex) = Arc::try_unwrap(writer)
    |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #53667 <https://github.com/rust-lang/rust/issues/53667> for more information

error[E0658]: `let` expressions in this position are unstable
   --> /root/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/grz-check-0.2.0/src/checker.rs:482:12
    |
482 |         && let Ok(mut writer_guard) = mutex.into_inner()
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #53667 <https://github.com/rust-lang/rust/issues/53667> for more information

For more information about this error, try `rustc --explain E0658`.
error: could not compile `grz-check` (bin "grz-check") due to 2 previous errors
```

--- a/packages/grz-check/Cargo.toml
+++ b/packages/grz-check/Cargo.toml
@@ -2,6 +2,7 @@
 name = "grz-check"
 version = "0.2.0"
 edition = "2024"
+rust-version = "1.88"
 license = "MIT"
 description = "grz-check is a tool for validating incoming files of Modellvorhaben §64e submissions to Genomrechenzentren (GRZ) in Germany."
 homepage = "https://github.com/BfArM-MVH/grz-tools/tree/main/packages/grz-check"

